### ISSUE TITLE
Feedback for: /getting-started/your-first-multi-screen-site/index.html

### ISSUE THREAD

**Issue by peterk** [google/WebFundamentals#619]
The img element in this code example should have an alt text, e.g. 

``` html
<div><img src="/images/awesome-photo.jpg" alt="You're Awesome!"></div>
```
